src/Raven.Client/Json/Converters/BlittableJsonConverter.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Sparrow.Json;
+
+namespace Raven.Client.Json.Converters
+{
+
+    internal sealed class BlittableJsonConverter : RavenTypeJsonConverter<BlittableJsonReaderObject>
```

Comment:
Those converters can be moved to Server.Json.Converters, I think?

Reply:
done